src/loaders/svg/tvgSvgLoader.cpp

```diff
@@ -900,6 +900,15 @@ static void _postpone(Array<SvgNodeIdPair>& nodes, SvgNode *node, char* id)
 }
 
 
+static void _postpone(Inlist<SvgNodeIdPair>& nodes, SvgNode *node, char* id)
```

Comment:
There’s no point in separating the function. `_postpone()` is neither self-explanatory nor reusable, and it adds unnecessary complexity with this change.